cypress.config.ts

```diff
@@ -11,4 +12,5 @@ export default defineConfig({
   },
   viewportWidth: 1920,
   viewportHeight: 1080,
+  videoCompression: false,
```

Comment:
I noticed we don't use this setting on Spruce, did you notice the videos were looking bad on the dashboard without it?

Reply:
Yes they looked pretty blurry when i was initially testing.